package.json

```diff
@@ -94,6 +95,7 @@
     "json2lcov": "^1.0.0",
     "mocha": "1.17.1",
     "moment": "^2.18.1",
+    "phantomjs": "1.9.7-15",
```

Comment:
What is the reason phantomjs is now needed as a dependency?

Reply:
To keep working in NodeJS 4.

Reply:
@davidffrench folllowing the answers. 

1. Do you know where browserify is used in this sdk? With such a large bump in the major version, I would be conscious about breaking changes.

The browsefly is used to minify the files which is the following task when npm install is executed. 

```
Running "browserify:dist" (browserify) task
found current version = 3.0.3
Version to inject is 3.0.3
>> Bundle dist/feedhenry.js created.
```
2. There should be license changes with these dependency updates also.
I will check it.

Reply:
@davidffrench  I updated the first comment with all answers.

Reply:
Perfect, sounds good to me @camilamacedo86